cron/sendUnfinishedCronWarnings.js

```diff
@@ -0,0 +1,34 @@
+const ERR = require('async-stacktrace');
+
+const logger = require('../lib/logger');
+const opsbot = require('../lib/opsbot');
+const sqldb = require('@prairielearn/prairielib/sql-db');
+const sqlLoader = require('@prairielearn/prairielib/sql-loader');
+
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+module.exports = {};
+
+module.exports.run = (callback) => {
+    if (!opsbot.canSendMessages()) return callback(null);
+    sqldb.query(sql.select_unfinished_cron_jobs, [], (err, result) => {
+        if (ERR(err, callback)) return;
+        let msg;
+        if (result.rowCount <= 0) {
+            msg = `_All cron jobs successfully finished_\n`;
```

Comment:
Probably no need to send this if we don't have any problems?

Reply:
Done.